helix-term/src/commands.rs

```diff
@@ -1923,7 +1923,8 @@ mod cmd {
         args: &[&str],
         _event: PromptEvent,
     ) -> anyhow::Result<()> {
-        let path = args.get(0).context("wrong argument count")?;
+        ensure!(!args.is_empty(), "wrong argument count");
+        let path = args.join(" ");
```

Comment:
Shouldn't we teach the arg parser to deal with quotes instead? I can imagine using `:open` to open multiple files

Reply:
I reworked it to support doublequotes for the filenames with space(s), and opening multiple files at once.